storybookjs/storybook
# Fixing README.md example for addons/info

Issue:

## What I did

Fixed a minor syntax error in addon/info/README.md

## How to test

Markdown error no testing needed.

Does this need a new example in the kitchen sink apps?
No

Does this need an update to the documentation?
No

If your answer is yes to any of these, please make sure to include it in your PR.
